alerting/provider/gitea/gitea.go

```diff
@@ -0,0 +1,162 @@
+package gitea
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"code.gitea.io/sdk/gitea"
+	"github.com/TwiN/gatus/v5/alerting/alert"
+	"github.com/TwiN/gatus/v5/config/endpoint"
+)
+
+// AlertProvider is the configuration necessary for sending an alert using Discord
+type AlertProvider struct {
+	RepositoryURL string `yaml:"repository-url"` // The URL of the Gitea repository to create issues in
+	Token         string `yaml:"token"`          // Token requires at least RW on issues and RO on metadata
+
+	// DefaultAlert is the default alert configuration to use for endpoints with an alert of the appropriate type
+	DefaultAlert *alert.Alert `yaml:"default-alert,omitempty"`
+
+	// SkipVerify disables SSL certificate verification
+	SkipVerify bool `yaml:"skip-verify,omitempty"`
```

Comment:
For consistency, this should be replaced by `client.Config`, which would enable `alerting.gitea.client.insecure` to achieve the same goal.

See https://github.com/TwiN/gatus/blob/dd435a8eafa604b0cb5ed9355adb17fdca48aba7/alerting/provider/googlechat/googlechat.go#L20 for an example of another provider that uses it.

Reply:
Sounds Good. @TwiN

Reply:
See https://github.com/TwiN/gatus/pull/842/commits/f456cf82c49cf5c34003ed9e0e695eaa0dfa2910